webtty/webtty.go

```diff
@@ -88,13 +91,14 @@ func (wt *WebTTY) Run(ctx context.Context) error {
 	go func() {
 		errs <- func() error {
 			buffer := make([]byte, wt.bufferSize)
+			line := ""
```

Comment:
I'm not sure why you want to define `line` here.

Reply:
Because a line is a combination of user‘s input multiple times. Before the user type "Enter", they should be stored temporarily.